isotope/service/pkg/srv/executable.go

```diff
@@ -25,13 +25,16 @@ import (
 
 	multierror "github.com/hashicorp/go-multierror"
 
-	"fortio.org/fortio/log"
-
+	"istio.io/pkg/log"
 	"istio.io/tools/isotope/convert/pkg/graph/script"
 	"istio.io/tools/isotope/convert/pkg/graph/svctype"
 	"istio.io/tools/isotope/service/pkg/srv/prometheus"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
If you do a rev of this PR, you may consider adding a comment here that random number usage in this code is not for a cryptographic purpose. rand.Seed() (may or should) trigger some various security linters. These annoying linters don't add much value, but are required by various CISO (chief information security officer) peeps for no good reason. There may be multiple CISOs in one firm...